js/bootstrap-notify.js

```diff
@@ -37,6 +37,9 @@
     else
       this.$note.addClass('alert-success');
 
+    if(this.options.class)
+      this.$note.addClass(this.options.class);
```

Comment:
Can we use `options.className` rather than `class` to avoid any possible parser issues (with `class` being a reserved keyword)